net/udphp-client/Makefile

```diff
@@ -0,0 +1,45 @@
+# Copyright (C) 2022 Almaz Gaifullin <gargargar@yandex.com>
+#
+# This is free software, licensed under the MIT
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=udphp-client
+PKG_VERSION:=0.0.1
```

Comment:
this version seems unused? can be removed.

Reply:
fixed